assets/components/src/popover/style.scss

```diff
@@ -0,0 +1,70 @@
+/**
+ * Popover
+ */
+
+@import '../../../shared/scss/colors';
+@import '~@wordpress/base-styles/colors';
+
+.newspack-popover {
+	.components-popover__content {
+		border: 1px solid $light-gray-500;
+		border-radius: 4px;
+		box-shadow: 0 4px 8px rgba( black, 0.08 );
+
+		.newspack-button {
+			font-weight: normal;
+			padding: 8px;
+
+			&:focus:not( :disabled ) {
+				box-shadow: inset 0 0 0 2px $primary-500;
+			}
+
+			&.is-link {
+				border-radius: 3px;
+				color: $dark-gray-primary;
```

Comment:
I'm getting this error, any idea why?

```
ERROR in ./assets/components/src/popover/style.scss
Module build failed (from ./node_modules/mini-css-extract-plugin-with-rtl/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):

				color: $dark-gray-primary;
          ^
      Undefined variable: "$dark-gray-primary".
```

Reply:
I guess you need to run `npm install`. I updated `@wordpress/base-styles` to the latest version, where `$dark-gray-primary` is included

Reply:
Switched to `$dark-gray-900` in https://github.com/Automattic/newspack-plugin/pull/515/commits/20d08b338a9af85e87a8f60a27cb88de106d58c9